src/sass/datetimepicker.scss

```diff
@@ -0,0 +1 @@
+@import "datetimepicker"
```

Comment:
Why isn't the contents of this file just the following?
+@import "variables";
+@import "core";